samza/jobs-common/src/main/java/org/sunbird/validator/MessageValidator.java

```diff
@@ -5,24 +5,24 @@
 
 import org.sunbird.common.exception.ProjectCommonException;
 import org.sunbird.common.responsecode.ResponseCode;
-import org.sunbird.common.responsecode.ResponseMessage;
 import org.sunbird.models.Constants;
 
 public class MessageValidator {
 
   public void validateMessage(Map<String, Object> message) {
-    if (!message.containsKey(Constants.OBJECT_TYPE)) {
-      ProjectCommonException.throwClientErrorException(ResponseCode.mandatoryParamsMissing, Constants.OBJECT_TYPE);
-    } else if (!message.containsKey(Constants.OPERATION_TYPE)) {
-      ProjectCommonException.throwClientErrorException(ResponseCode.mandatoryParamsMissing, Constants.OPERATION_TYPE);
-    } else if (!message.containsKey(Constants.EVENT)) {
-      ProjectCommonException.throwClientErrorException(ResponseCode.mandatoryParamsMissing, Constants.EVENT);
-    } else if (!message.containsKey(Constants.IDENTIFIER)) {
+
+    validate(message, Constants.IDENTIFIER, ResponseCode.mandatoryParamsMissing);
+    validate(message, Constants.OBJECT_TYPE, ResponseCode.mandatoryParamsMissing);
+    validate(message, Constants.OPERATION_TYPE, ResponseCode.mandatoryParamsMissing);
```

Comment:
Validate supported operation (UPSERT, DELETE) and event (transactional, instructional) types.